radis/extractions/templates/extractions/_query_generation_section.html

```
@@ -0,0 +1,37 @@
+                    <p class="mb-0 mt-2">
+                        <small>Using AI to create a search query from your extraction fields...</small>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+{% elif generated_query %}
+    {# Generation already ran for these fields (e.g. re-render after a #}
+    {# validation error or back navigation); show the result statically #}
+    {# instead of regenerating and overwriting the user's query. #}
+    <div id="query-generation-section">
```

HTMLHint (1.9.2)

[error] The id value [ query-generation-section ] must be unique.

(id-unique)